cpacker/MemGPT
# Allow overriding config location with `MEMGPT_CONFIG_PATH`

**Please describe the purpose of this pull request.**
Is it to add a new feature? Is it to fix a bug?

**How to test**
Run `export MEMGPT_CONFIG_PATH=/some/path/config` and make sure memgpt still works. 

**Have you tested this PR?**
Yes. `memgpt run` works normally. 

**Related issues or PRs**
 #329

**Is your PR over 500 lines of code?**
No
